Snakefile

```diff
@@ -1,141 +1,163 @@
 import glob
 import os
 configfile: "./CONFIG.txt"
-chromosomes=config["chromosomes"]
-genome=config["genome_name"]
-
-array_positions = glob.glob(os.path.join("resources/snp_arrays", genome, "SNP_array_positions.txt"))[0]
 
-print(array_positions)
+chromosomes=config["chromosomes"]
+output_dir = config["output_dir"]
+temp_dir = config["temp_dir"] if config["temp_dir"] else output_dir + "/temp"
 
-cases=config["bams"].keys()
 
 rule all:
     input:
```

Comment:
I did drop the case 'id', and only considered a single tumour / normal pair at a time.

Partly because our cases are usually analyzed in terms of multiple biopsies.  eg. POG965 biop2 vs POG965 biop3.
There isn't always a simple id logic.

Reply:
For POG I used modified snakefiles that had a format like output/{case}/{tumour_id}_{normal_id}/ but I suppose for GSC use it will likely be run once per case anyways so this solution is probably fine